src/cnd/cnd.ts

```diff
@@ -277,23 +289,53 @@ export class Cnd {
     return info.listen_addresses;
   }
 
+  /**
+   * Initiate a swap request on cnd.
+   *
+   * @param swap The details of the swap to initiate.
+   * @returns The URL of the swap request on the cnd REST API.
+   * @throws Problem | Error A {@link Problem} from the cnd REST API or an Error.
+   */
   public async postSwap(swap: SwapRequest): Promise<string> {
     const response = await this.client.post("swaps/rfc003", swap);
 
     return response.headers.location;
   }
 
+  /**
+   * List the swaps handled by this cnd instance.
+   *
+   * @returns An Array of {@link SwapSubEntity}, which contains details of the swaps.
+   * @throws Problem | Error A {@link Problem} from the cnd REST API or an Error.
+   */
   public async getSwaps(): Promise<SwapSubEntity[]> {
     const response = await this.fetch("swaps");
     const entity = response.data as Entity;
 
     return entity.entities as SwapSubEntity[];
   }
 
+  /**
+   * Fetch data from the REST API
+   *
+   * @param path The URL to GET.
+   * @returns The data returned by cnd.
+   * @throws Problem | Error A {@link Problem} from the cnd REST API or an Error.
+   */
   public fetch<T>(path: string): AxiosPromise<T> {
     return this.client.get(path);
   }
 
+  /**
+   * Proceed with an action request on the cnd REST API.
+   * This request could be a GET HTTP request, which only returns data needed to further proceed with the action,
+   * or a POST request, which is the action itself, to instruct cnd about a swap
+   *
+   * @param action The action to perform
```

Comment:
```suggestion
   * @param action The action to perform.
```

Reply:
@da-kami Hum, why are you fixing typos at a 10% review? 🤔